docs/DeveloperGuide.md

```diff
@@ -1598,8 +1598,129 @@ testers are expected to do more *exploratory* testing.
+            * Client Name: Alex Yeoh
+            * Client Phone: 87438807
+            * Client Address: Blk 30 Geylang Street 29. #06-40
+            * Recipe Name: Chicken Rice
+            * Recipe Ingredients: Rice x 200 g, Chicken x 1 whole
+            * Order Price: $4.00
+            * Order Deadline: 12 October 2021 at 12 pm
+            * Order Quantity: 1
+            * Completion status: Completed
+    1. Test case: `done-o 1`
+       <br>Expected: The first order is marked as completed. It's position in the Order list may change. Details of
+           marked order is shown in the result display.
+    1. Test case: `done-o 2`
+       <br>Expected: The second order is still marked as completed. Details of marked order is shown in the result display.
+
+1. Marking an order as undone
+    1. Prerequisites: There are exactly 2 orders in the order list.
+        1. First order has the following details:
+            * Client Name: Bernice Yu
+            * Client Phone: 98762159
+            * Client Address: Blk 12 Holland Village Street 1. #01-40
+            * Recipe Name: Nasi Lemak
+            * Recipe Ingredients: Rice x 200 g, Fish x 1 whole
+            * Order Price: $3.00
+            * Order Deadline: 15 December 2021 at 1 pm
+            * Order Quantity: 2
+            * Completion status: Uncompleted
+        1. Second order has the following details:
+            * Client Name: Alex Yeoh
+            * Client Phone: 87438807
+            * Client Address: Blk 30 Geylang Street 29. #06-40
+            * Recipe Name: Chicken Rice
+            * Recipe Ingredients: Rice x 200 g, Chicken x 1 whole
+            * Order Price: $4.00
+            * Order Deadline: 12 October 2021 at 12 pm
+            * Order Quantity: 1
+            * Completion status: Completed
+    1. Test case: `undone-o 2`
+       <br> The second order is marked as uncompleted. It's position in the Order list may change. Details of
+       marked order is shown in the result display.
+    1. Test case: `undone-o 1`
+       <br>Expected: The first order is still marked as completed. Details of marked order is shown in the result display.
+
 ### Recipe
 
+1. Adding a Recipe
+    1. Prerequisites (should be satisfied for each individual test case):
+        1. Recipe list does not contain a recipe with the following details:
+            * Recipe Name: Chicken Rice
+            * Recipe Ingredients: Rice x 200 g, Chicken x 1 whole
+            * Recipe Price: $4.00
+    1. Test case: `add-r rn/Chicken Rice ri/Rice-200-g, Chicken-1-Whole rp/4`
+        <br> Expected: A new recipe is added to the recipe list with the given details. Details of the added
+        recipe are shown in the result display box.
+    1. Test case: `add-r rn/Chicken Rice ri/Rice-200-g, Chicken-1-Whole`
+       <br> Expected: No recipe is added. Error details shown in the result display box.
+    1. Test case: `add-r rn/Chicken Rice rp/4`
```

Comment:
This is a valid command!!!!

Please check all your commands if it is valid. If this kind of bugs are reported in the PE, we are really shooting ourselves in the foot.